packages/nodes-base/credentials/MessageBirdApi.credentials.ts

```diff
@@ -0,0 +1,14 @@
+import { ICredentialType, NodePropertyTypes } from 'n8n-workflow';
+
+export class MessageBirdApi implements ICredentialType {
+	name = 'messageBirdApi';
+	displayName = 'MessageBird API';
+	properties = [
+		{
+			displayName: 'API Key',
+			name: 'accessKey',
+			type: 'string' as NodePropertyTypes,
```

Comment:
I don't understand why we need to cast `'string'` as `NodePropertyTypes` since this is just [an alias for various string literals](https://github.com/n8n-io/n8n/blob/c7a901083b30611ed03f403dae1473d8ad716e8e/packages/workflow/src/Interfaces.ts#L386) including `'string'`. But I do realize that's how it's being done in the rest of the codebase.